engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41400to41500.java

```diff
@@ -140,9 +140,9 @@ private void updateSystemVmTemplates(final Connection conn) {
 
         final Map<Hypervisor.HypervisorType, String> newTemplateChecksum = new HashMap<Hypervisor.HypervisorType, String>() {
             {
-                put(Hypervisor.HypervisorType.KVM, "81b3e48bb934784a13555a43c5ef5ffb");
-                put(Hypervisor.HypervisorType.XenServer, "1b178a5dbdbe090555515340144c6017");
-                put(Hypervisor.HypervisorType.VMware, "e6a88e518c57d6f36c096c4204c3417f");
+                put(Hypervisor.HypervisorType.KVM, "0d95bb3d9385097dec8b485e46eae34b");
```

Comment:
Note: after 4.15 is cut; we need to create a new upgrade path from 4.15->4.16 and move these there. cc @Pearl1594